Sources/SwifterAuth.swift

```diff
@@ -125,6 +125,18 @@ public extension Swifter {
         }
         
         let urlScheme = "swifter-\(client.consumerKey)"
+        authorizeSSO(withCallback: urlScheme, success: success, failure: failure)
+    }
+  
+    func authorizeSSO(withCallback urlScheme: String,
```

Comment:
Instead of duplicating the method signature with a customizable callback URL, this should use an optional callback. 

```swift
func authorizeSSO(
    withCallbackURLScheme urlScheme: String? = nil, 
    success: SSOTokenSuccessHandler, 
    failure: FailureHandler? = nil
) {
    guard let client = client as? SwifterAppProtocol else { 
        // handle error
        return
    } 
    let urlScheme = urlScheme ?? "swifter-\(client.consumerKey)"
    // ...
}